service_mesh/v2x/ossm-troubleshooting-istio.adoc

```diff
@@ -0,0 +1,68 @@
+[id="ossm-troubleshooting"]
+= Troubleshooting your service mesh
+include::modules/ossm-document-attributes.adoc[]
+:context: troubleshooting-ossm
+
+toc::[]
+
+This section describes how to identify and resolve common problems in {ProductName}. Use the following sections to help troubleshoot and debug problems when deploying {ProductName} on {product-title}.
+
+// The following include statements pull in the module files that comprise the assembly.
+
+include::modules/ossm-understanding-versions.adoc[leveloffset=+1]
+
+== Troubleshooting Operator installation
+
+In addition to the information in this section, be sure to review the following topics:
+
+* xref:../../operators/understanding/olm-what-operators-are.adoc[What are Operators?].
+
+* xref:../../operators/understanding/olm/olm-understanding-olm.adoc[Operator Lifecycle Management concepts].
+
+* xref:../../support/troubleshooting/troubleshooting-operator-issues.adoc[OpenShift Operator troubleshooting section].
+
+* xref:../../support/troubleshooting/troubleshooting-installations.adoc[OpenShift installation troubleshooting section]
+
+include::modules/ossm-validating-operators.adoc[leveloffset=+2]
+
+include::modules/ossm-troubleshooting-operators.adoc[leveloffset=+2]
+
+== Troubleshooting the control plane
+
+The Service Mesh _control plane_ is composed of *Istiod* which consolidates several previous control plane components (Citadel, Galley, Pilot) into a single binary. Deploying the `ServiceMeshControlPlane` also creates the other components that make up {ProductName} as described in the xref:../../service_mesh/v2x/ossm-architecture.adoc[architecture] topic.
+
+include::modules/ossm-validating-smcp.adoc[leveloffset=+2]
+
+include::modules/ossm-accessing-kiali.adoc[leveloffset=+3]
+
+include::modules/ossm-accessing-jaeger.adoc[leveloffset=+3]
+
+include::modules/ossm-troubleshooting-smcp.adoc[leveloffset=+2]
+
+== Troubleshooting the data plane
+
+The _data plane_ is a set of intelligent proxies that intercept and control all inbound and outbound network communications between services in the service mesh.
+
+{ProductName} relies on a proxy sidecar within the application’s pod to provide service mesh capabilities to the application.
+
+include::modules/ossm-troubleshooting-injection.adoc[leveloffset=+2]
+
+For more information about sidecar injection, see xref:../../prepare-to-deploy-applications-ossm.adoc#ossm-automatic-sidecar-injection_deploying-applications-ossm[Enabling automatic injection]
+
+include::modules/ossm-troubleshooting-proxy.adoc[leveloffset=2]
+
+For more information about troubleshooting pod issues, see xref:../../support/troubleshooting/investigating-pod-issues.adoc[OpenShift pod troubleshooting]
```

Comment:
```suggestion
For more information about troubleshooting pod issues, see xref:../../support/troubleshooting/investigating-pod-issues.adoc[Investigating pod issues].
```
I'd use the actual names of the pages so the link doesn't say something different than the heading it goes to